packages/adaptivecards-tools-sdk/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "adaptivecards-tools",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./lib/index.js",
+  "types": "types/index.d.ts",
+  "scripts": {
+    "clean": "rimraf lib types",
+    "clean-build": "npm run clean && npm run build",
+    "build": "rollup -c"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "rollup-plugin-typescript2": "^0.25.3",
+    "@rollup/plugin-commonjs": "^13.0.2",
```

Comment:
I'm curious on what's rollup library for dev dependency?

Reply:
for pack the package